File: Stepik Course/Task4.py
from selenium import webdriver
import time
from selenium.webdriver.support.ui import Select

try:
    browser = webdriver.Chrome(r'C:\Users\kate_s\Downloads\chromedriver\chromedriver.exe')
    browser.get("http://suninjuly.github.io/selects1.html")

   
    num1 = int(browser.find_element_by_id("num1").text)
    num2 = int(browser.find_element_by_id("num2").text)

    dr = browser.find_element_by_css_selector("select#dropdown")
    dropdown = Select(browser.find_element_by_xpath('//*[@id="dropdown"]'))
    dropdown.select_by_value(str(num1 + num2))
    
  
    button = browser.find_element_by_css_selector('[type = "submit"]')
    button.click()


finally:
    time.sleep(30)
    browser.quit()

File: Stepik course tasks in Java/Task4.java
package com.company;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.support.ui.Select;

import java.util.concurrent.TimeUnit;


public class Main {


    public static void main(String[] args) throws InterruptedException {

        System.setProperty("webdriver.chrome.driver", "C:\\Users\\kate_s\\Downloads\\chromedriver\\chromedriver.exe"); //define path to the webdriver
        WebDriver driver = new ChromeDriver(); //   Create instance of ChromeDriver interface

    try {
        driver.get("http://suninjuly.github.io/selects1.html");

        int num1 = Integer.parseInt(driver.findElement(By.id("num1")).getText());
        int num2 = Integer.parseInt(driver.findElement(By.id("num2")).getText());

        Select dropdown = new Select(driver.findElement(By.xpath("//*[@id=\"dropdown\"]")));
        dropdown.selectByValue(String.valueOf((num1 + num2)));

        WebElement button = driver.findElement(By.cssSelector("[type = \"submit\"]"));
        button.click();
    }

finally
        {
            TimeUnit.SECONDS.sleep(6);
            driver.quit();
        }
    }
}
